sql/expression/function/user_test.go

```diff
@@ -9,25 +9,26 @@ import (
 )
 
 func TestUser(t *testing.T) {
```

Comment:
This should probably go in some other file now that you've deleted the non-test file.